dev/mima

```diff
@@ -24,24 +24,63 @@ set -e
 FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
 cd "$FWDIR"
 
-echo -e "q\n" | build/sbt oldDeps/update
-rm -f .generated-mima*
+# All default projects except for assemblies:
+SPARK_PROJECTS=(
+  "common/network-common"
+  "common/network-shuffle"
+  "common/network-yarn"
+  "common/sketch"
+  "common/tags"
+  "common/unsafe"
+  "core"
+  "docker-integration-tests"
+  "examples"
+  "external/akka"
+  "external/flume"
+  "external/flume-sink"
+  "external/kafka"
+  "external/mqtt"
+  "external/twitter"
+  "external/zeromq"
+  "extras/kinesis-asl"
```

Comment:
It turns out that we don't support MiMa for the Maven build anyways, so I can just safely pull this in from one of SBT's `fullClasspath` outputs.